### DISCUSSION TITLE
Error (function() { // If GPC on, set DOM

### DISCUSSION THREAD

**Discussion by grantiago** [community/community#23968] [Other Feature Feedback, Questions, & Ideas]
<p>Is it just me? In the rss feed of my github page, before the xml declaration there is a long line of code:</p>
<blockquote>
<p>function() { // If GPC on, set DOM property to true if not already true if (true) { if<br>
(navigator.globalPrivacyControl) return Object.defineProperty(navigator, ‘globalPrivacyControl’, { value: true, enumerable: true }) } else { // If GPC off, set DOM property prototype to false so it may be overwritten // with a true value by user agent or other extensions if (typeof navigator.globalPrivacyControl !== “undefined”) return Object.defineProperty(Object.getPrototypeOf(navigator), ‘globalPrivacyControl’, { value: false, enumerable: true }) } // Remove script tag after execution document.currentScript.parentElement.removeChild(document.currentScript) })()</p>
</blockquote>
<p>Which would make the xml invalid. and a quick search I see it on some other pages as well.</p>
<p>thanks.</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/grantiago">@grantiago</a>, welcome to the GitHub Support Community! Can you post a link to an example if this happening?</p>

**Comment by grantiago** [comment]
<p>Hello. thanks for getting back to me. I’ve poked around there looking for the cause. Nothing.</p>
<p>I’ll warn you up front, I have mucked around in that repo with no experience with Jekyll. I have a clone on my server here at home – ubuntu 16.04 and the unformatted code is showing there as well. So I’m guessing it is an error on my part and not a github error. But, please have a look.</p>
<p>Here’s the link <a href="https://grantiago.github.io/feed.xml" rel="noopener nofollow ugc">https://grantiago.github.io/feed.xml</a></p>
<p>It’s from a public repo so you can have a look at that as well:</p>
<p><a href="https://github.com/grantiago/grantiago.github.io" rel="noopener nofollow ugc">https://github.com/grantiago/grantiago.github.io</a></p>

**Comment by grantiago** [comment, marked as answer]
<p>Whoops. 99 percent sure it is  the duck duck go privacy extension in chrome. Turned them all off and it went away.</p>
<p>And then back on one at a time until the error returned. It’s tough to be 100% certain with the caching in Jekyll.</p>
